source/install/yum.rst

```diff
@@ -18,50 +18,55 @@ of the following distributions:
    :local:
 
 Package Contents
-================
+================================================================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - Package
+     - Contains
+   * - percona-server-mongodb-40
```

Comment:
from what I know on centos packages also don't have major version inside package name so "-40" should be removed